### ISSUE TITLE
Alt text - Needs support ASAP

### ISSUE THREAD

**Issue by Nicholas-Arthur-Cook** [react-icons/react-icons#584]
### Discussed in https://github.com/react-icons/react-icons/discussions/583

<div type='discussions-op-text'>

<sup>Originally posted by **Nicholas-Arthur-Cook** July 19, 2022</sup>
Hi react-icons team. I'm currently messing around with your icons on one of my webpages, and I noticed that there is no way to specify the alt text attribute for the images in the icons. This leads to a screen reader just reading "image" for each icon, with a screen reader user having no way to identify what the icon itself does. 

This is an **extremely important issue** because it means that every single site using these icons is not WCAG compliant, because all images are required to have alt text. 
The way that I was planning on using the alt text for the icon was to say what the button itself would do, or give it the null alt text attribute using alt="".  Please let me know if there is something wrong with how I am using the icons, or if there is a way I could contribute to the project with a PR myself.

Thanks!
Nick</div>

**Comment by lipe-dev** [comment]
just add an aria-label bro.

**Comment by albertoncp** [comment]
Hello @Nicholas-Arthur-Cook , react-icons is a library that encompasses many libraries such as fontawesome or grommet, for that reason, the problem comes from the adjacent libraries  , however, every icon uses svg, wich doesn't have an alt atributte, what's more similar is aria-label, as you see , grommet library does have an aria-label atributte.

https://github.com/grommet/grommet-icons/blob/master/src/js/StyledIcon.js#L50

<img width="890" alt="Screenshot_3" src="https://user-images.githubusercontent.com/104977482/216810826-6ba529bb-37b4-49fc-9fb0-509f501de1ac.png">

**Comment by staff0rd** [comment]
Also there is `title` which appears to have similar functionality as `alt` considering hovering the svg results in a tooltip with the `title` in it.

**Comment by AccessiT3ch** [comment]
@kamijin-fanta - Can we make the `title` prop required to ensure accessibility of the icons?

**Comment by Nicholas-Arthur-Cook** [comment]
Thanks all!
